### ISSUE TITLE
pip install pyMOR fails when git is not installed

### ISSUE THREAD

**Issue by sdrave** [pymor/pymor#297]
See the following error message:

```
Collecting pymor
  Downloading pymor-0.4.1.tar.gz (275kB)
    100% |████████████████████████████████| 276kB 2.4MB/s 
    Complete output from command python setup.py egg_info:
    fatal: Not a git repository (or any parent up to mount point /tmp)
    Stopping at filesystem boundary (GIT_DISCOVERY_ACROSS_FILESYSTEM not set).
    Traceback (most recent call last):
      File "/tmp/pip-build-bkjvoilh/pymor/setup.py", line 102, in write_version
        '--tags', '--candidates', '20', '--match', '*.*.*']).strip()
      File "/usr/lib/python3.5/subprocess.py", line 626, in check_output
        **kwargs).stdout
      File "/usr/lib/python3.5/subprocess.py", line 708, in run
        output=stdout, stderr=stderr)
    subprocess.CalledProcessError: Command '['git', 'describe', '--tags', '--candidates', '20', '--match', '*.*.*']' returned non-zero exit status 128
```

I have just checked the tarball, there _is_ a `version.py` inside, so I really do not understand why git gets called. @renemilk, could you take a look?

**Comment by sdrave** [comment]
Ok, so the reason is that I have tried to install pyMOR 0.4 with Python 3. However, I do not really understand why `version.py` fails to import. Maybe we should add a Python version check to the 0.4.x branch?

**Comment by renemilk** [comment]
You're thinking in ```setup.py```? Should work.

**Comment by sdrave** [comment]
I just check that the classifiers in setup.py of 0.4.x specifically say Python 2.7. So strictly speaking this is at best an issue of pip. Shall we close this?

**Comment by renemilk** [comment]
I don't think pip is supposed to, or claiming to, care about the trove classifiers.

**Comment by renemilk** [comment]
This should fix itself with #294
